From patches Mon Sep 17 00:00:00 2001
From: Laszlo Ersek <lersek@redhat.com>
To: aliguori@us.ibm.com, seabios@seabios.org, qemu-devel@nongnu.org
Cc: 
Date: Mon,  8 Apr 2013 15:13:22 +0200
Message-id: 1365426803-1781-5-git-send-email-lersek@redhat.com
Subject: [PATCH 4/5] hw/acpi: export acpi_checksum()

Again, this enables reuse when preparing per-table fw_cfg blobs later.

Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 hw/acpi.h |    2 ++
 hw/acpi.c |    2 +-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/hw/acpi.h b/hw/acpi.h
index e3e17e9..2c89e59 100644
--- a/hw/acpi.h
+++ b/hw/acpi.h
@@ -167,4 +167,6 @@ typedef struct acpi_table_std_header {
 } QEMU_PACKED AcpiTableStdHdr;
 
 extern const AcpiTableStdHdr acpi_dfl_hdr;
+
+int acpi_checksum(const uint8_t *data, int len);
 #endif /* !QEMU_HW_ACPI_H */
diff --git a/hw/acpi.c b/hw/acpi.c
index f24be53..07634e1 100644
--- a/hw/acpi.c
+++ b/hw/acpi.c
@@ -69,7 +69,7 @@ static void acpi_register_config(void)
 
 machine_init(acpi_register_config);
 
-static int acpi_checksum(const uint8_t *data, int len)
+int acpi_checksum(const uint8_t *data, int len)
 {
     int sum, i;
     sum = 0;
-- 
1.7.1



_______________________________________________
SeaBIOS mailing list
SeaBIOS@seabios.org
http://www.seabios.org/mailman/listinfo/seabios
